plugin-build/src/main/kotlin/io/sentry/android/gradle/AndroidComponentsConfig.kt

```diff
@@ -238,3 +241,18 @@ private fun AndroidComponentsExtension<*, *, *>.configureVariants(callback: (Var
         onVariants70(this, callback)
     }
 }
+
+private fun getReleaseInfo(project: Project, variant: Variant): ReleaseInfo {
+    val appExtension = project.extensions.getByType(AppExtension::class.java)
+    var applicationId =
+        appExtension.defaultConfig.applicationId ?: appExtension.namespace.toString()
+    var versionName = appExtension.defaultConfig.versionName ?: "1.0.0"
```

Comment:
```suggestion
    var versionName = appExtension.defaultConfig.versionName ?: "undefined"
```
Maybe that is a bit more explicit than `1.0.0`? Idk if we have server-side validation of semver though, but I'd try to send undefined and see what happens

Reply:
After implementing the backend, we don't have a semver validation here so we're I guess we're free to pass in what makes sense in that case. I assume `undefined` seems reasonable if a version can't be determined.